components/sensor/atm90e32.rst

```diff
@@ -0,0 +1,139 @@
+household. Three current measurements are read via CT clamps. 
+
+The CircuitSetup Split Single Phase Real Time Whole House Energy Meter can read 2 current channels and one voltage channel.
+.. figure:: images/atm90e32-cs-2chan-full.jpg
+    :align: center
+    :width: 50.0%
+
+    CircuitSetup Split Single Phase Real Time Whole House Energy Meter.
+
+The CircuitSetup Expandable 6 Channel ESP32 Energy Meter can read 6 current channels and 2 voltage channels at a time, this board has two atm90e32 ICs and requires two sensors to be configured in ESPHome.
+.. figure:: images/atm90e32-cs-6chan-full.jpg
+    :align: center
+    :width: 50.0%
+
+    CircuitSetup Expandable 6 Channel ESP32 Energy Meter Main Board.
+
+The IC supports a number of measurements, these are not yet exported by ESPHome
+Electrical parameters measurement: Vrms, Irms, mean active/reactive/apparent
+power, frequency, power factor and phase angle.
+Enhanced event detection: sag, over voltage, phase loss, over current, reverse V/I
+phase sequence, calculated neutral line current INC over-current and frequency
+upper and lower threshold.
+
+.. code-block:: yaml
+
+    # Example CS 2-channel entry
+    sensor:
+      - platform: atm90e32
+        cs_pin: 5
+        voltage_a:
+          name: "EMON Line Voltage A"
+        current_a:
+          name: "EMON CT1 Current"
+        current_b:
+          name: "EMON CT2 Current"
+        power:
+          name: "EMON Total Active Power CT1-2"
+        frequency:
+          name: "EMON Line Frequency"
+        line_freq: 50Hz
+        gain_pga: 2X
+        gain_volt_a: 47660
+        gain_ct_a: 12577
+        gain_ct_b: 12577
+        update_interval: 60s
+
+    # Example CS 6-channel entry
+    sensor:
+      - platform: atm90e32
+        cs_pin: 5
+        voltage_a:
+          name: "EMON Line Voltage A"
+        current_a:
+          name: "EMON CT1 Current"
+        current_b:
+          name: "EMON CT2 Current"
+        current_c:
+          name: "EMON CT3 Current"
+        power:
+          name: "EMON Total Active Power CT1-3"
+        frequency:
+          name: "EMON Line Frequency"
+        line_freq: 50Hz
+        gain_pga: 2X
+        gain_volt_a: 47660
+        gain_ct_a: 12577
+        gain_ct_b: 12577
+        gain_ct_c: 12577
+        update_interval: 60s
+      - platform: atm90e32
+        cs_pin: 4
+        current_a:
+          name: "EMON CT4 Current"
+        current_b:
+          name: "EMON CT5 Current"
+        current_c:
+          name: "EMON CT6 Current"
+        power:
+          name: "EMON Total Active Power CT4-6"
+        line_freq: 50Hz
+        gain_pga: 2X
+        gain_volt_a: 47660
+        gain_ct_a: 12577
+        gain_ct_b: 12577
+        gain_ct_c: 12577
+        update_interval: 60s
+
+.. note::
+
+    The gain values must be calibrated for each installation, see instructions at https://github.com/CircuitSetup/Split-Single-Phase-Energy-Meter#calibration. 
```

Comment:
The link here is for their library and assumes the use of their logging format. ESPHome users will be confused by that. Please add a quick section below (potentially referencing the link) that explains calibration of the sensor with ESPHome's `calibrate_linear` sensor filter (see CT clamp docs as example).